x/auth/keeper/migrations.go

```diff
@@ -63,7 +63,7 @@ func (m Migrator) Migrate3to4(ctx sdk.Context) error {
 // set the account without map to accAddr to accNumber.
 //
 // NOTE: This is used for testing purposes only.
-func (m Migrator) V45_SetAccount(ctx sdk.Context, acc types.AccountI) error {
+func (m Migrator) V45SetAccount(ctx sdk.Context, acc types.AccountI) error {
```

Comment:
We actually always name our migration `V{version}_Action` as it is more readable than simple camel case. (e.g. `govv046.Migrate_V046_6_To_V046_7`)